rpcs3/Emu/RSX/VK/VulkanAPI.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#define VK_USE_PLATFORM_WIN32_KHR
```

Comment:
I guess we can add support later.
I think Vulkan backend still needs works (as every others ones) but it's likely easier if it's in sync with master branch.